docs/pages/includes/common-faq.mdx

```diff
@@ -0,0 +1,25 @@
+{{ level="##" }}
+{{ level }} Do we use Enterprise or Open Source Teleport for connecting resources to our Teleport Cluster?
+
+Teleport Team, Enterprise, and Enterprise cloud clusters should use the Enterprise release of the 
+`teleport` binary for agents. Running the non-Enterprise release could result in
```

Comment:
In this page we've used "Open Source", "Community", and "non-Enterprise" to describe the same thing. Can we pick a single term and use it consistently?

Reply:
@ptgott thoughts?

Reply:
Updated to this.